.github/workflows/shared-publish-docker-versioned.yaml

```diff
@@ -31,6 +31,10 @@ on:
         description: If true, will skip tests when packaging JAR. Defaults to false. Set to true for test-only repos.
         type: boolean
         default: false
+      vulnerability_severity:
+        description: The severity to fail the workflow if such vulnerability is detected. DO NOT override it unless a Jira ticket is raised.
```

Comment:
I would add the options here so people know what is available. Such as CRITICAL,HIGH,MEDIUM (don't know the exact terms)